lib/Doctrine/Adapter/Statement/Oracle.php

```diff
@@ -314,7 +314,7 @@ public function fetch($fetchStyle = Doctrine_Core::FETCH_BOTH, $cursorOrientatio
                 return oci_fetch_array($this->statement, OCI_NUM + OCI_RETURN_NULLS + OCI_RETURN_LOBS);
             break;
             case Doctrine_Core::FETCH_OBJ:
-                return oci_fetch_object($this->statement, OCI_NUM + OCI_RETURN_NULLS + OCI_RETURN_LOBS);
+                return oci_fetch_object($this->statement);
```

Comment:
This function only accepts 1 parameter (https://secure.php.net/oci_fetch_object) I assume this was copy/paste from lines above.